### PR TITLE
Revert "B3 Groups + related fixes"

### PR DESCRIPTION
**Pull request by czue**
Reverts dimagi/commcare-hq#11760

(checking if this fixes http://manage.dimagi.com/default.asp?228613 locally)

### CODE REVIEW

File: corehq/apps/users/forms.py [hunk 1]
```diff
@@ -527,50 +488,41 @@ def users_form(self):
             form.fields['selected_ids'].choices = self.all_users
             return form
 
-        <form class="form disable-on-submit" id="edit_users" action="" method='post'>
-            <legend>{% trans 'Specify Users At This Location' %}</legend>
-            {% crispy users_per_location_form %}
-        </form>
+        # template.html
+        <script src="{% static 'hqwebapp/js/ui-element.js' %}"></script>
+        <script src="{% static 'hqwebapp/js/lib/jquery-ui/jquery-ui-1.9.2.multiselect-deps.custom.min.js' %}"></script>
```
Comment (dimagimon) [inline corehq/apps/users/forms.py]: E501 line too long (119 > 115 characters)

**Comment by czue** [comment]
@biyeun this does fix http://manage.dimagi.com/default.asp?228613 - do you want to take a look at doing a fix before we revert? i poked around for a bit and couldn't figure it out.

**Comment by orangejenny** [comment]
If this does get merged, the references to ui-element.js may need to be updated, due to https://github.com/dimagi/commcare-hq/pull/11878 being merged.

**Comment by biyeun** [comment]
merge this instead https://github.com/dimagi/commcare-hq/pull/11902 :+1:

**Pull request closed without merging**